app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -446,7 +446,7 @@ def center_toolbar_filename_classic
               ems_container ems_middleware container_project container_route container_replicator container_image
               ems_network security_group floating_ip cloud_subnet network_router
               container_image_registry ems_infra flavor host container_build
-              ontap_file_share ontap_logical_disk container_topology middleware_topology
+              ontap_file_share ontap_logical_disk container_topology middleware_topology middleware_server
```

Comment:
Perhaps I'm taking as assumption that the magnitude order of merging a PR in master by MiQ core team is high in time compared with other branch where we can work in provider related tasks.
A practical scenario would be:
- A is coding a feature for example, to add controllers to live metrics approach.
- B is coding an angular front end and needs those controllers.
So, there is a dependency between boths PR. So, if B needs that A's PR is merged in master prior to consolidate the work, the cycle can be long. 
In essence my suggestion is to use a separate branch for these specific scenarios, and once A  and B are consolidated, to make a single PR into master.
Of course, this is only needed when are dependencies, I know that normal tasks should be more isolated and independent.
Anyway, in github there are several ways to get same goal.
Perhaps, creating a clone with a single branch, and consolidating from there is the same.
And also, if master can review and merge with a reasonable rhythmn all of this discussion is outdated.